src/tests/integration/test_nycdb.py

```diff
@@ -125,6 +125,22 @@ def test_hpd_complaints(conn):
     assert row_count(conn, 'hpd_complaints') == 100
 
 
+def test_dof_exemptions(conn):
+    drop_table(conn, 'dof_exemptions')
+    drop_table(conn, 'dof_exemption_classification_codes')
+    hpd_complaints = nycdb.Dataset('dof_exemptions', args=ARGS)
+    hpd_complaints.db_import()
+    assert row_count(conn, 'dof_exemptions') == 10
+
+
+def test_dof_exemption_classification_codes(conn):
+    drop_table(conn, 'dof_exemptions')
+    drop_table(conn, 'dof_exemption_classification_codes')
+    hpd_complaints = nycdb.Dataset('dof_exemptions', args=ARGS)
```

Comment:
```suggestion
    dof_exemption_classification_codes = nycdb.Dataset('dof_exemption_classification_codes', args=ARGS)
```
Should use the classification codes dataset here to make sure we're testing both datasets.

Reply:
So, my understanding is we actually want to keep the name of the dataset `dof_exemptions` as that's the key name of the larger "dataset" in nycdb that includes the two tables. I get this error when I try and chang the name to `dof_exemption_classification_codes`, which speaks to this issue: 
```
self = <nycdb.dataset.Dataset object at 0x7f6616f0b850>
dataset_name = 'dof_exemption_classification_codes'
args = namespace(user='nycdb', password='nycdb', host='db', database='nycdb_test', port='5432', root_dir='/nycdb/src/tests/integration/data', hide_progress=False)

    def __init__(self, dataset_name, args=None):
        self.name = dataset_name
        self.args = args
        self.db = None
    
        if self.args:
            self.root_dir = self.args.root_dir
        else:
            self.root_dir = './data'
    
>       self.dataset = datasets()[dataset_name]
E       KeyError: 'dof_exemption_classification_codes'

nycdb/dataset.py:46: KeyError
```

Reply:
That being said, I think we are still able to test the `dof_exemption_classification_codes` table in the `assert clause, given that both tables in the dataset get imported by the `.db_import()` command:
```
dof_exemptions = nycdb.Dataset('dof_exemptions', args=ARGS)
dof_exemptions.db_import()
assert row_count(conn, 'dof_exemption_classification_codes') == 10
   ```

Reply:
Ahh, understood. Looks fine then! Thanks!